### ISSUE TITLE
`Contrib` folder missing

### ISSUE THREAD

**Issue by ale94mleon** [kuelumbus/rdkit-pypi#51]
Hello, awesome work making our lives easy with this library! I am struggled trying to use `RDConfig.RDContribDir` in my project.

I create this function to import sascorer (it is no so nice, I know :-) )
```python
def import_sascorer():
    # In order to import sascorer from RDConfig.RDContribDir
    from rdkit.Chem import RDConfig
    import os, importlib.util as importlib_util
    spec=importlib_util.spec_from_file_location('sascorer', os.path.join(RDConfig.RDContribDir, 'SA_Score', 'sascorer.py'))
    sascorer = importlib_util.module_from_spec(spec)
    spec.loader.exec_module(sascorer)
    return sascorer
```

When I use `conda install rdkit`, everything works fine. But when I use `python -m pip install rdkit` (as recomended in [PyPi](https://pypi.org/project/rdkit/)). I get:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 7, in import_sascorer
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1016, in get_code
  File "<frozen importlib._bootstrap_external>", line 1073, in get_data
FileNotFoundError: [Errno 2] No such file or directory: '/home/ale/anaconda3/envs/test/lib/python3.10/site-packages/rdkit/Contrib/SA_Score/sascorer.py'
```
And indeed the file doesn’t exist. So, how could I install the contribute directory via pip? A lot of thanks in advance!

**Comment by SGenheden** [comment]
Can this be re-opened? Or have you ruled out including the Contrib folder in the wheels, @kuelumbus ?

At least I think it should be clearly documented what the wheels does not include, as of now I assume that I get the same thing with pip and conda installations (but I don't).

**Comment by kuelumbus** [comment]
Yes, you are right. The next version should include the `contrib` directory.

**Comment by kuelumbus** [comment]
This is fixed in version `2022.3.5`.
